test/misc/update.js

```diff
@@ -0,0 +1,13 @@
+"use strict";
+const jsdom = require("../..");
+
+exports["Update option url"] = t => {
+  const window = jsdom.jsdom("", { url: "http://example.com/" }).defaultView;
+
+  t.strictEqual(window.location.href, "http://example.com/");
+
+  jsdom.changeURL(window, { url: "http://localhost/" });
+  t.strictEqual(window.location.href, "http://localhost/");
+
+  t.done();
+};
```

Comment:
Add a test that it throws a TypeError if you pass it an invalid URL (like `"asdf"`).